dts/arm/st/u5/stm32u535.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025 Harris Tomy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/u5/stm32u5.dtsi>
+
+/ {
+	soc {
+		/* USB-C PD is not available on this part. */
+		/delete-node/ ucpd@4000dc00;
+
+		/* Advanced Encryption Standard HW accelerator is not available on this part. */
+		/delete-node/ aes@420c0000;
+
+		compatible = "st,stm32u535", "st,stm32u5", "simple-bus";
+
+		usb: usb@40016000 {
+			compatible = "st,stm32-usb";
+			reg = <0x40016000 0x400>;
+			interrupts = <73 0>;
+			interrupt-names = "usb";
+			num-bidir-endpoints = <8>;
+			ram-size = <2048>;
+			maximum-speed = "full-speed";
+			clocks = <&rcc STM32_CLOCK(APB2, 24) 0x01000000>,
```

Comment:
remove the 3rd value: macro already `STM32_CLOCK()` expands to 2 values: bus ID + clock enable bit mask.

Reply:
done